new_tests/test_extract.py

```diff
@@ -203,11 +195,21 @@ def test_n_parsed(self, n_parsed_files):
         parsed_files = (len(p._parsed_mscx), len(p._parsed_tsv))
         assert parsed_files == n_parsed_files
 
-    def test_check(self, caplog, get_ignored_warnings):
+    @pytest.fixture()
+    def ignored_warnings(self):
+        return {"ms3.Parse.mixed_files.Did03M-Son_regina-1762-Sarti.mscx": [(2, 94)],
+                 "ms3.Parse.mixed_files.BWV_0815.mscx": [(1, 1, 40, 85, 97, 131, 139)]}
+
+    def test_parse_ignored_warnings_file(self, ignored_warnings):
+        assert self.parsed_parse_obj.parse_ignored_warnings(os.path.join(os.path.expanduser("~"),
```

Comment:
Please replace computation of the `unittest_metacorpus` directory by the fixture `directory`. All fixtures from conftest are automatically imported.